boards/stm32l496g-disco/include/arduino_board.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C)  2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_stm32l496g-disco
+ * @{
+ *
+ * @file
+ * @brief       Board specific configuration for the Arduino API
+ *
+ * @author      Alexandre Abadie  <alexandre.abadie@inria.fr>
```

Comment:
Copy-paste

Reply:
Yes, I left it by intention since you are the author ;-)